src/services

```diff
@@ -18,7 +18,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"pmInfoDAO",                   "lazy"
 p({"class":"foam.nanos.boot.NSpec", "name":"scriptDAO",                   "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.GUIDDAO(new foam.nanos.script.ScriptRunnerDAO(new foam.dao.PMDAO(new foam.dao.JDAO(foam.nanos.script.Script.getOwnClassInfo(), \"scripts\"))));", "client":"{\"of\":\"foam.nanos.script.Script\"}"})
 p({"class":"foam.nanos.boot.NSpec", "name":"cronDAO",                     "lazy":true,  "serve":true,  "serviceScript":"return new foam.dao.GUIDDAO(new foam.nanos.script.ScriptRunnerDAO(new foam.dao.PMDAO(new foam.dao.JDAO(foam.nanos.cron.Cron.getOwnClassInfo(), \"cronjobs\"))));","client":"{\"of\":\"foam.nanos.cron.Cron\"}"})
 p({"class":"foam.nanos.boot.NSpec", "name":"localUserDAO",                "lazy":true,  "serve":false, "serviceScript":"return new foam.dao.PMDAO(new foam.dao.SequenceNumberDAO (new foam.dao.JDAO(foam.nanos.auth.User.getOwnClassInfo(), \"users\")));"})
-p({"class":"foam.nanos.boot.NSpec", "name":"userDAO",                     "lazy":true,  "serve":true,  "serviceScript":"return /* new foam.nanos.auth.AuthenticatedUserDAO(*/x.get(\"localUserDAO\")/*)*/;","client":"{\"of\":\"foam.nanos.auth.User\", \"cache\":true}"})
+p({"class":"foam.nanos.boot.NSpec", "name":"userDAO",                     "lazy":true,  "serve":true,  "serviceScript":"return new net.nanopay.auth.email.EmailVerificationUserDAO(x.get(\"localUserDAO\"));","client":"{\"of\":\"foam.nanos.auth.User\",\"cache\":true,\"decorators\": [{\"class\":\"foam.blob.BlobServiceDecorator\",\"of\":\"foam.nanos.auth.User\"}]}"})
```

Comment:
"of" is no longer required on the BlobServiceDecorator decorator.

Reply:
Removed